link_shortener/view_routes.py

```diff
@@ -104,6 +104,15 @@ async def delete_link_view(request, user, link_id):
                         status_code=str(status)
                     ), status=status)
 
+@view_blueprint.route('/delete/confirm/<link_id>', methods=['GET'])
+@login_required
+@credential_whitelist_check
+async def confirm_delete_link_view(request, user, link_id):
+    return html(template_loader(
+                    template_file='delete_link.html',
+                    payload=link_id,
+                    status_code=200))
```

Comment:
Payload klidně pojmenuj ID když je tam jen id. Payload je takové generické když tam toho cpeš víc.

Jinak status_code pak v templatu nikde nepoužíváš, tak ho tam ani neposílej. Ale můžeš přidat status za závorku, to je informace pro server aby zobrazil ten kód uživateli po requestu. Tady klidně těch 200 jak tam máš.

Reply:
return html(template_loader(
                    template_file='delete_link.html',
                    link_id=link_id), status=200)